### DISCUSSION TITLE
color icons not displayed in iOS AR Mode

### DISCUSSION THREAD

**Discussion by Arumugamhns** [google/model-viewer#1942] [Q&A]
I have creating a object and change the color of the object. 

That color icons are displayed in AR mode (Android chrome 86)

But not displayed in iOS chrome and safari. is there any script needed to add resolve this
![Capture](https://user-images.githubusercontent.com/60291301/101473752-39933080-3970-11eb-800e-f4859a071a7c.JPG)

**Comment by elalish** [comment]
DOM overlay and the rest of model-viewer's API is only available on the web, so for AR you need webXR mode to have these features. Since WebXR is not yet supported on iOS, we have to open Apple's QuickLook app with the USDZ or .reality file you provide in `ios-src`. Until Apple supports WebXR, we have no way to support our API in AR on iOS.

**Reply by sprengerst** [reply]
Do yo know if there are any plans from Apple to get rid of USDZ?

**Reply by LeoFA123** [reply]
Have you heard anything on this? How to add annotations to an AR model

**Reply by sprengerst** [reply]
@LeoFA123 you have to use .reality files.
